README.md

```diff
@@ -1 +1,188 @@
-# React-Capstone-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <br/>
+
+  <h3><b>React Capstone Project </b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+-   [📖 About the Project](#about-project)
+    -   [🛠 Built With](#built-with)
+        -   [Tech Stack](#tech-stack)
+        -   [Key Features](#key-features)
+-   [💻 Getting Started](#getting-started)
+    -   [Prerequisites](#prerequisites)
+    -   [Setup](#setup)
+    -   [Install](#install)
+    -   [Usage](#usage)
+    -   [Run tests](#Run-tests)
+-   [🎥 Live Demo](#live-demo)
+-   [🎥 Future Feature](#future-feature)
+-   [👥 Authors](#authors)
+-   [🎥 Project Presentation](#Project-presentation)
```

Comment:
- [ ] Kindly record a video presenting the features of the project you built, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Once%20you%20have,very%20proud%20of.) requirement.
- [ ] Please make sure to deploy your app and put the link to the live demo in the readme, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20project%20should,your%20README%20file.) requirement.